kapre/backend.py

```diff
@@ -212,52 +212,26 @@ def mu_law_decoding(signal_mu, quantization_channels):
     return signal
 
 
-def freq_mask(self, input_f, param, name='freq_mask'):
-    """Apply masking to a spectrogram in the freq domain.
-    TensorFlow/io
-    
+def random_masking_along_axis(input_f, param, axis, name='masking'):
```

Comment:
as in the example i prototyped (
```
def random_masking_along_axis(x, axis, x_min=0, x_max=None, max_width=None): 
  if x_max is None:
    x_max = x.shape[axis]
  if max_width is None:
    max_width = (x_max - x_min) // 4  # 4 is an arbitrary choice though
  # do the work..
```)
I think this `param` should be unpacked and listed in this API.